actionview/lib/action_view/renderer/partial_renderer.rb

```diff
@@ -439,6 +439,11 @@ def collection_with_template
 
         content = template.render(view, locals)
         content = layout.render(view, locals) { content } if layout
+
+        if template.eligible_for_collection_caching?(as: as)
```

Comment:
I'm not sure we need to cache fixed template renders here actually.
Thinking about this some more I'm scared we might be doing multiple writes, which is... not good :grin:

--
Kasper

> Den 01/04/2015 kl. 05.36 skrev Rafael Mendonça França <notifications@github.com>:
> 
> In actionview/lib/action_view/renderer/partial_renderer.rb:
> 
> > @@ -439,6 +439,11 @@ def collection_with_template
> >  
> >          content = template.render(view, locals)
> >          content = layout.render(view, locals) { content } if layout
> > +
> > +        if template.eligible_for_collection_caching?(as: as)
> We changed collecton_with_template so maybe we need a test?
> 
> —
> Reply to this email directly or view it on GitHub.
>